src/layout/header/DesktopHeader.tsx

```diff
@@ -47,7 +49,11 @@ export default function DesktopHeader() {
             <MenuItem {...menu} className="menu-item" as="a" href="#">
               Previous Orders
             </MenuItem>
-            <MenuItem {...menu} className="menu-item rounded-b-lg text-left">
+            <MenuItem
+              {...menu}
+              className="menu-item rounded-b-lg text-left"
+              onClick={() => signOut()}
```

Comment:
you can usually just pass the function to avoid creating an anonymous function, but maybe typescript could complain depending of the prop types
```suggestion
              onClick={signOut}
```

Reply:
Nice, thanks!

Reply:
Typescript indeed does complain 😅